source/etherpad.rst

```diff
@@ -0,0 +1,194 @@
+.. highlight:: console
+
+.. sidebar:: About
+
+  This guide has been tested with Uberspace 7.1.1 and Etherpad Lite 1.6.3
+
+#############
+Etherpad Lite
+#############
+
+Etherpad Lite is a real-time collaborative writing tool. It is based on NodeJs and comes with lot of possible plugins.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * Node.js_ and its package manager npm_ 
+  * MySQL_ 
+  * supervisord_
+  * domains_
+
+Prerequisites
+=============
+
+We're using Node.js_ in the stable version 8:
+
+::
+
+ [etherpad@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [etherpad@stardust ~]$ 
+
+You'll need your MySQL_ credentials. Get them with ``my_print_defaults``:
+
+::
+
+ [etherpad@stardust ~]$ my_print_defaults client
+ --default-character-set=utf8mb4
+ --user=etherpad
+ --password=MySuperSecretPassword
+ [etherpad@stardust ~]$ 
+
+Your domain needs to be setup:
+
+::
+
+ [etherpad@stardust ~]$ uberspace web domain list
+ etherpad.uber.space
+ [etherpad@stardust ~]$ 
+
+Installation
+============
+
+First get the Etherpad Lite source code from Github_:
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ git clone https://github.com/ether/etherpad-lite ~/etherpad
+
+Then run the etherpad script, to install the dependencies:
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ ~/etherpad/bin/installDeps.sh
+
+Configuration
+=============
+
+Configure port
+--------------
+
+Since Node.js applications use their own webserver, you need to find a free port and bind your application to it.
+
+.. include:: includes/generate-port.rst
+
+Set up a Database
+-----------------
+
+Run the following code to create the database ``<username>_etherpad``:
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ mysql -e "CREATE DATABASE $USER_etherpad COLLATE utf8mb4_unicode_ci;"
```

Comment:
Doesn't work for me:

```
[moritz@stardust ~]$ mysql -e "CREATE DATABASE $USER_etherpad COLLATE utf8mb4_unicode_ci;"
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'COLLATE utf8mb4_unicode_ci' at line 1
```

Reply:
`CREATE DATABASE ${USER}_etherpad COLLATE utf8mb4_unicode_ci;`. Otherwise bash tries to evaluate `$USER_etherpad`.